chez-libs/unison/primops.ss

```diff
@@ -101,13 +101,35 @@
     unison-POp-VALU
     unison-POp-VWLS
 
+    unison-POp-UPKB
+    unison-POp-ADDI
+    unison-POp-DIVI
+    unison-POp-EQLI
+    unison-POp-MODI
+    unison-POp-LEQI
+    unison-POp-POWN
+    unison-POp-VWRS
```

Comment:
these POps aren't strictly necessary as part of this PR, but they were needed to run the tests I was doing.